packages/react-components/react-utilities/src/compose/resolveShorthand.ts

```diff
@@ -26,11 +27,17 @@ export const resolveShorthand: ResolveShorthandFunction = (value, options) => {
 
   let resolvedShorthand = {} as UnknownSlotProps;
 
-  if (typeof value === 'string' || typeof value === 'number' || Array.isArray(value) || isValidElement(value)) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  if (typeof value === 'string' || typeof value === 'number' || Array.isArray(value) || isValidElement<any>(value)) {
```

Comment:
Is it really? Does not seem to throw without `<any>`...

Reply:
This has nothing to do with this PR btw, this is happening on `master` already. seems like some changes have happened on some of our dependencies and although it passes on our `build` and `type-check` it fails on local vscode ts version, which is visually unpleasant, but it's not a big deal though.

![image](https://user-images.githubusercontent.com/5483269/229500454-4478b9b8-91d9-4ca2-b050-bf18bb67143f.png)